package.json

```diff
@@ -56,7 +59,7 @@
     "enzyme": "^2.8.2",
     "eslint-plugin-babel": "^4.1.1",
     "eslint-plugin-jsx": "0.0.2",
-    "eslint-plugin-react": "^7.0.1",
```

Comment:
Any particular reason for downgrading?

Reply:
All the additions in package.json were made due to several warnings after `npm install` saying that all these dependencies need to be installed

Reply:
Can you please post a screenshot? I'm unsure if they are used anywhere in the app. :smile:

Reply:
Of the warnings?

Reply:
You can just make a branch that is synched with upstream and post ss?

Reply:
Thank you, I'll do the needful
![screen shot 2018-01-31 at 10 45 15 am](https://user-images.githubusercontent.com/29275810/35606241-f7a8eed0-0673-11e8-83fa-7f96ae109979.png)

Reply:
Thanks so much for posting, you can probably make an object in package.json and list these as [peer dependencies](https://nodejs.org/en/blog/npm/peer-dependencies/) :+1: